style.css

```diff
@@ -72,12 +82,307 @@ body {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  padding: 0;
+  background-image: url(img/Snapshoot\ Portfolio.png);
+  background-repeat: no-repeat;
+  background-size: 80% 40%;
+  background-position-x: center;
+  background-position-y: 15px;
+}
+
+.mainContainer2 {
+  width: 100vw;
+  height: auto;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  padding: 0;
+  background-image: url(img/Snapshoot\ Portfolio2.png);
+  background-repeat: no-repeat;
+  background-size: 80% 40%;
+  background-position-x: center;
+  background-position-y: 15px;
+}
+
+.mainContainer3 {
+  width: 100vw;
+  height: auto;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  padding: 0;
+  background-image: url(img/Snapshoot\ Portfolio3.png);
+  background-repeat: no-repeat;
+  background-size: 80% 40%;
+  background-position-x: center;
+  background-position-y: 15px;
+}
+
+.mainContainer4 {
+  width: 100vw;
+  height: auto;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  padding: 0;
+  background-image: url(img/Snapshoot\ Portfolio4.png);
+  background-repeat: no-repeat;
+  background-size: 80% 40%;
+  background-position-x: center;
+  background-position-y: 15px;
+}
+
+.infoCard {
+  margin-top: 80%;
+}
+
+.name {
+  font-weight: 700;
+  font-size: 32px;
+  color: #172b4d;
+  margin-bottom: 5%;
+}
+
+.decription {
+  width: 90vw;
+  height: auto;
+  font-size: 15px;
+}
+
+.job {
+  list-style: none;
+}
+
+.info_job {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  display: flex;
+  justify-content: flex-start;
+  gap: 5%;
+  width: 90vw;
+  height: auto;
+  margin-bottom: 5%;
+}
+
+.linkButtons {
+  display: flex;
+  padding: 4px 0;
+  justify-content: flex-start;
+  gap: 5%;
+  margin-top: 5%;
+  font-size: 15px;
+  color: #6070ff;
+  border: none;
+  border-radius: 8px;
+  text-decoration: none;
+  margin-bottom: 5%;
+}
+
+.link {
+  background-color: #ebebff;
+  font-size: 15px;
+  color: #6070ff;
+  border: none;
+  border-radius: 8px;
+  text-decoration: none;
+  padding: 5px 10px;
+}
+
+.more {
+  size: 20px;
+  line-height: 24px;
+  font-size: 17px;
+  padding: 2% 5%;
+  color: #396df2;
+  border-radius: 8px;
+  border: 1px solid #396df2;
+  background-color: white;
+  margin-top: 5%;
+  margin-bottom: 2%;
+}
+
+.more:hover {
+  background-color: #7e9dec;
+}
+
+.more:active {
+  background-color: #396df2;
+  box-shadow: 0 5px #666;
+  transform: translateY(4px);
+}
+
+/* Portfolio "about me" section */
+
+.mainContainerAboutme {
+  box-sizing: border-box;
+  width: 100vw;
+  height: auto;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  margin-bottom: 10%;
+  padding: 0;
+  margin-top: 5rem;
```

Comment:
- [ ] [OPTIONAL] Kindly give your about section a ` border-top-right-radius` as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205436931-e008ee13-0a11-4717-9616-89ec6096c3f1.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205436939-fd81b627-6d5f-48da-83af-6bf6ed4e17fd.png)